packages/smart-contracts/hardhat.config.ts

```diff
@@ -1,7 +1,12 @@
 import '@typechain/hardhat';
 import '@nomiclabs/hardhat-waffle';
-import '@nomicfoundation/hardhat-verify';
```

Comment:
Because the zk plugin few lines below, `@matterlabs/hardhat-zksync-verify` is also importing this one, and extending it for zksync type networks.
And we get a compilation error from hardhat about a duplicate task.
So this one is still available in the app.